docs/docs/documentation/getting-started/installation/backend-config.md

```diff
@@ -16,7 +16,7 @@
 | TZ                            |          UTC          | Must be set to get correct date/time on the server                                  |
 | ALLOW_SIGNUP<super>\*</super> |         false         | Allow user sign-up without token                                                    |
 | LOG_CONFIG_OVERRIDE           |                       | Override the config for logging with a custom path                                  |
-| LOG_LEVEL                     |         info          | logging level configured                                                            |
+| LOG_LEVEL                     |         info          | logging level (e.g. critical, error, warning, info, debug, trace)                   |
```

Comment:
I don't think Python actually supports "trace" out of the box: https://docs.python.org/3/library/logging.html#levels

```suggestion
| LOG_LEVEL                     |         info          | logging level (e.g. critical, error, warning, info, debug)                          |
```

Reply:
Wow I'm blind, thanks!